File: aoj/volume2/n0216/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        while (true) {

            int waterUsed = sc.nextInt();

            if (waterUsed == -1) {
                break;
            }

            int cost = 1150;

            if (10 < waterUsed) {
                cost += Math.min((waterUsed - 10) * 125, 1250);
            }

            if (20 < waterUsed) {
                cost += Math.min((waterUsed - 20) * 140, 1400);
            }

            if (30 < waterUsed) {
                cost += (waterUsed - 30) * 160;
            }

            System.out.println(4280 - cost);

        }

    }

}

File: aoj/volume2/n0216/Main.cpp
#include <iostream>

using namespace std;

int main() {

    while (true) {

        int waterUsed;
        cin >> waterUsed;

        if (waterUsed == -1) {
            break;
        }

        int cost = 1150;

        if (waterUsed > 30) {
            cost += (waterUsed - 30) * 160;
            waterUsed -= waterUsed - 30;
        }

        if (waterUsed > 20) {
            cost += (waterUsed - 20) * 140;
            waterUsed -= waterUsed - 20;
        }

        if (waterUsed > 10) {
            cost += (waterUsed - 10) * 125;
        }

        cout << 4280 - cost << endl;

    }

    return 0;

}
